server/lib/realtime/message_dispatcher.ex

```diff
@@ -0,0 +1,46 @@
+# This file draws from https://github.com/phoenixframework/phoenix/blob/9941711736c8464b27b40914a4d954ed2b4f5958/lib/phoenix/channel/server.ex
+# License: https://github.com/phoenixframework/phoenix/blob/518a4640a70aa4d1370a64c2280d598e5b928168/LICENSE.md
+
+defmodule Realtime.MessageDispatcher do
+  alias Phoenix.Socket.Broadcast
+
+  @doc """
+  Hook invoked by Phoenix.PubSub dispatch.
+  """
+  def dispatch(subscribers, _from, %Broadcast{payload: payload} = msg) do
```

Comment:
we can extend this later to include sending to channel pid depending on channel intercepts